app/views/prototypes/edit.html.haml

```diff
@@ -14,20 +14,20 @@
     .row
       .col-md-12
         = f.fields_for :prototype_images, (@prototype.prototype_images.main.first) do |main_image|
-          .cover-image-upload.main_image_preview
+          .cover-image-upload.main_image_preview{style: "background-image: url(#{@prototype.prototype_images.main.first.image});" "width: 100%"}
```

Comment:
Line is too long. [144/80]<br>Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace